### PR TITLE
:rotating_light: Housekeeping (unused variables)

### PR DESCRIPTION
**Pull request by AndreMiras**
Fixes linter no-unused-vars mainly and removes some other dead code.
Mainly addresses the components/ folder.

### CODE REVIEW

File: components/cdpActiveTable/cdpActiveTable.js [hunk 1]
```diff
@@ -101,29 +87,8 @@ EnhancedTableHead.propTypes = {
...
-        }
-      : {
-          color: theme.palette.text.primary,
-          backgroundColor: theme.palette.secondary.dark,
-        },
-  title: {
-    flex: '1 1 100%',
-  },
-}));
-
```
Comment (AndreMiras) [inline components/cdpActiveTable/cdpActiveTable.js:126]: Seems surprising, but all theses functions removed were actually not used, so good to clean

File: components/vaultAssetRow/vaultAssetRow.js [hunk 2]
```diff
@@ -22,525 +21,18 @@ import {
...
   DISABLE_COLLATERAL_LEND,
   DISABLE_COLLATERAL_LEND_RETURNED,
-  CONNECT_WALLET,
   LENDING_SUPPLY_RETURNED,
-  LENDING_BORROW_RETURNED,
 } from '../../stores/constants';
 
 import classes from './vaultAssetRow.module.css';
 
-function VaultAssetDetails({ vault, account }) {
```
Comment (AndreMiras) [inline components/vaultAssetRow/vaultAssetRow.js:32]: Definitely not used from within the file, and not exported so good to let it go

Comment (antonnell) [reply components/vaultAssetRow/vaultAssetRow.js:32]: Yea, was in the orginal build, but got scrapped later on.

**Comment by antonnell** [comment]
Thanks for making the changes!
Appreciate somebody cleaning up the mess that is this code.

**Pull request merged**